pepper-apis/docs/specification/src/endpoints/studies.invitation-check.yml

```diff
@@ -0,0 +1,35 @@
+post:
+  operationId: checkInvitationStatus
+  tags:
+    - Invitations
+  summary: CheckInvitationStatus
+  description: Check the status of an invitation, whether it exits and is valid or not.
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          required:
+            - auth0ClientId
+            - auth0Domain
+            - invitationId
+            - recaptchaToken
+          properties:
+            auth0ClientId:
+              description: the id of the client
+              type: string
+            auth0Domain:
+              description: the domain of the client
+              type: string
+            invitationId:
+              description: the id of the invitation
+              type: string
+            recaptchaToken:
+              description: the user's reCAPTCHA token
+              type: string
```

Comment:
Zip code?

Reply:
Don't know concretely what we're doing with zip codes yet. That will be in different PR.